### PR TITLE
:arrow_up::one::two: theme_kit

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

File: web_theme_kit/__manifest__.py [hunk 1]
```diff
@@ -5,9 +5,9 @@
     "name": """Brand kit""",
     "summary": """Brand your odoo instance in few clicks""",
     "category": "Debranding",
-    # "live_test_url": "http://apps.it-projects.info/shop/product/theme-kit?version=11.0",
+    # "live_test_url": "http://apps.it-projects.info/shop/product/theme-kit?version=12.0",
     "images": ['images/brandkit.png'],
-    "version": "11.0.1.1.2",
+    "version": "12.0.1.1.2",
```
Comment (ilmir-k) [inline web_theme_kit/__manifest__.py:9]: @GabbasovDinar why it is 1.**1**.2 while in the changelog the version is 1.0.2

**Comment by ilmir-k** [comment]
@GabbasovDinar @yelizariev 
We have strange travis error:
``web_theme_kit/__init__.py:1: [E7990(manifest-version), ] Wrong version value in module's __manifest__.py``

**Review by ilmir-k — APPROVED** [review]

**Comment by ilmir-k** [comment]
@Ramil-Mukhametzyanov don't forget to leave your review here

**Review by trojikman — APPROVED** [review]

**Comment by trojikman** [comment]
I approve to merge it now

**Pull request merged**